main/lsp/LSPPreprocessor.cc

```diff
@@ -340,8 +340,8 @@ LSPPreprocessor::canonicalizeEdits(u4 v, unique_ptr<DidCloseTextDocumentParams>
             openFiles.erase(localPath);
             // Use contents of file on disk.
             auto fileType = getFileType(localPath, config->opts);
-            edit->updates.push_back(
-                make_shared<core::File>(move(localPath), readFile(localPath, *config->opts.fs), fileType, v));
+            auto fileContents = readFile(localPath, *config->opts.fs);
+            edit->updates.push_back(make_shared<core::File>(move(localPath), move(fileContents), fileType, v));
```

Comment:
Good catch, was this from running clang-tidy?

Reply:
It looks like some tests had failed; I guessed these changes were required to fix failing tests.